### ISSUE TITLE
hello how ca i use this code to train a new datasets ? 

### ISSUE THREAD

**Issue by fikri2992** [sniklaus/pytorch-liteflownet#4]
hello it's really great work. but can i use this to train a new datasets ?

it's working on my GTX 1060 3GB in 30sec for 2 images 

it saves me a lot of time for my research

**Comment by fjchange** [comment]
I am afraid your GPU memory is too small to train it.
And this repo just transfers the models trained on caffe to pytorch, which means this repo have no training code.
This may satisfy you : https://github.com/twhui/LiteFlowNet

**Comment by sniklaus** [comment]
I am afraid that my time is limited and I do not have the resources to provide the training code. The provided implementation is fully differentiable though, which is why you just need to provide a data loader and implement a loss function.

As mentioned by @fjchange, the [official implementation](https://github.com/twhui/LiteFlowNet) provides a training portion that you might want to consider using instead. I also agree that three gigabytes might be too little for training.
